File: python/1.py
#!/usr/bin/python3

def twoSum(nums, target):
    i = 0
    j = 0
    l = len(nums)
    for i in range(0, l):
        for j in range(1, l):
            if nums[i] + nums[j] == target and i != j:
                return (i, j)


if __name__ == '__main__':
    nums = [2, 5, 5, 11]
    target = 10
    print(twoSum(nums, target))

File: go/1.go
package leetcode

func twoSum(nums []int, target int) []int {
	var idx1 int
	var idx2 int
	length := len(nums)
	for i := 0; i < length; i++ {
		for j := 1; j < length; j++ {
			if nums[i]+nums[j] == target && i != j {
				idx1 = i
				idx2 = j
			}
		}
	}
	return []int{idx1, idx2}
}
